jep/1/README.adoc

```diff
@@ -552,12 +552,16 @@ The reasoning should provide evidence of consensus within the
 community and discuss important objections or concerns raised
 during discussion.
 
-. **Backwards Compatibility** -
-  All JEPs must include a section describing any incompatibilities and their severity.
-  The JEP must explain how the sponsor proposes to deal with these incompatibilities.
-  If there are no dackwards compatibility concerns, the section must say that.
-  JEP submissions that do not adequately discuss backwards compatibility,
-  when such discussion is required will not be approved as JEP Drafts.
+. **Backwards Compatibility** - All JEPs must include a section describing any incompatibilities and their severity.
+  The JEP must explain how the it proposes to deal with these incompatibilities.
+  If there are no backwards compatibility concerns, the section must say that.
+. **Security** - All JEPs must include a section describing any security risks and their severity.
```

Comment:
more generally:

… describing its impact on the security of affected components, and, if any, how these are dealt with. This includes discussion on potential security issues and how they are prevented, as well as how the proposed changes interact with existing permissions, authentication, authorization, etc.